### DISCUSSION TITLE
MicroSD card connector

### DISCUSSION THREAD

**Discussion by JakubChalupnik** [erichelgeson/BlueSCSI-v1#221] [Q&A] [answered]
Folks,

what's the right MicroSD card connector? Is it Molex 104031-0811 or is that just a placeholder in KiCad?

I recall regular SD connectors were a mess so I'd better doublecheck before shelling out almost 15 EUR for S+H :)

Thanks a lot!

**Comment by jamesy0ung** [comment, marked as answer]
It is the 104031-0811. You are welcome to check the parts list in the wiki if you don't believe me.

**Reply by JakubChalupnik** [reply]
Well, I could not find the part number anywhere, all I could find was "Micro SD connector" and it turned out there are at least two different footprints, with this Molex being the rare one :) Thanks, I'll order some of the Molex ones.
